### DISCUSSION TITLE
how about to support multi database

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [diesel-rs/diesel#2982] [Ideas]
I want to connect to multi database right now, still did not found how to define multi schema files seperately. The database A schema will override database B schema.rs right now, what should I do to define a multi schema.rs files and I could connect to multit database in one diesel app? This is what I have already tried to do:https://stackoverflow.com/questions/70378560/is-it-possible-to-make-diesel-to-handle-multi-database

**Comment by weiznich** [comment]
As already answered [here](https://github.com/diesel-rs/diesel/issues/1728) thats something currently not supported by diesel cli, so asking again will not make the feature magically appear. 

That means you either need to wait till someone implements the feature or start implementing it on your own. Contributions are welcome.

**Comment by jiangxiaoqiang** [comment]
I am define multiple `schema.rs` and multiple `models.rs` with different toml config, when connect to database, get different connection. seems works fine. I think the diesel docs should add this way to docs to give a choice option with other people.
